### PR TITLE
Static bundle refactor and tests

### PR DESCRIPTION
**Pull request by rowanmanning**

### CODE REVIEW

File: lib/static-bundle.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+const fs = require('fs');
+const path = require('path');
+const pfs = require('q-io/fs');
+const uniqueid = require('./utils/uniqueid');
+const URL = require('url');
+
+const defaultStaticBundlesDirectory = path.resolve(__dirname, '../deprecated/static-bundles');
+
+function getStaticBundleStream(url, staticBundlesDirectory) {
```
Comment (AlbertoElias) [inline lib/static-bundle.js]: If this has now been moved out, would it make sense to put the file in the `deprecated` directory so it's easier to remove sometime in the future and also doesn't divert attention from the main functionality?

Comment (rowanmanning) [inline lib/static-bundle.js]: Good idea, I'll move it :)

File: test/unit/mock/express.mock.js [hunk 2]
```diff
@@ -4,5 +4,16 @@ const sinon = require('sinon');
 
 module.exports = sinon.stub().returns({
 	disable: sinon.spy(),
-	use: sinon.spy()
+	use: sinon.spy(),
```
Comment (AlbertoElias) [inline test/unit/mock/express.mock.js]: I thought lingering commas weren't proper JSON

Comment (rowanmanning) [inline test/unit/mock/express.mock.js]: Oops! Linter should probably pick that up

Comment (AlbertoElias) [inline test/unit/mock/express.mock.js]: Yes, it should. Don't know what might have gone wrong =/

**Comment by AlbertoElias** [comment]
Looks good :+1:

**Pull request merged**